modules/geochemistry/test/tests/interrogate_reactions/tests

```diff
@@ -0,0 +1,112 @@
+[Tests]
+  [./invalid_swap]
+    type = RunException
+    input = invalid_swap.i
+    # only run in opt mode, otherwise the Outputs/perf_graph=true in the CIVET debug tests triggers:
+    #Assertion `common_actions.size() == 1' failed
+    #Should be only one common_output Action
+    #at moose/framework/src/base/MooseApp.C, line 819
+    method = 'OPT'
+    expect_err = 'Matrix is not invertible, which signals an invalid basis swap'
+    threading = '!pthreads'
+    requirement = 'The geochemistry interrogator will throw an error for an invalid swap'
+    issues = '#14981'
+    design = 'GeochemicalModelInterrogator.md GeochemicalModelRoot.md'
+  [../]
+  [./gypsum]
+    type = 'RunApp'
+    input = 'gypsum.i'
+    allow_warnings = true
+    method = 'OPT'
+    threading = '!pthreads'
```

Comment:
Why are these only opt and !pthreads? Is the pthreads the same issue as the fluid properties user objects?

Reply:
Only run in opt mode, otherwise the Outputs/perf_graph=true in the CIVET debug tests triggers:
  Assertion `common_actions.size() == 1' failed
  Should be only one common_output Action

Not sure about pthreads.  I can check....